src/FeaturePlugin.php

```diff
@@ -16,6 +16,7 @@
 use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes_Woo_Subscriptions_Notes;
 use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes_Tracking_Opt_In;
 use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes_WooCommerce_Payments;
+use \Automattic\WooCommerce\Admin\Features\Navigation;
```

Comment:
We could wrap this in a feature flag too so unnecessary calculations aren't made.